app/exporters/patients_exporter.rb

```diff
@@ -10,7 +10,17 @@ def self.csv(patients)
       csv << csv_headers
 
       patients.in_batches(of: BATCH_SIZE).each do |batch|
-        batch.each do |patient|
+        batch.includes(
+          :registration_facility,
+          :phone_numbers,
+          :address,
+          :medical_history,
+          :prescription_drugs,
+          :latest_bp_passports,
+          { latest_scheduled_appointments: :facility },
```

Comment:
I assume we're cascadingly including `facility` with `latest_scheduled_appointments` and `latest_blood_pressures` because of patient transfer and/or just patient visiting elsewhere?

Reply:
The latest scheduled appointments pull in their facility to power the new requirement in this PR: Showing the follow-up facility name.

The latest BPs pull in their facility because the latest BP's facility info is included in the CSV as well.